### DISCUSSION TITLE
However, non-members should not add items to their cart.

### DISCUSSION THREAD

**Discussion by Osman06TR** [opencart/opencart#12738] [Q&A]
I am using Openvart Version 4.0.2.3. (I'm learning)
Non-members cannot see prices and I have adjusted them.
However, non-members should not add items to their cart. I cannot make adjustments.
How and in which module can I do it?

Thanks.

**Comment by mhcwebdesign** [comment]
Best to post this question on the OpenCart forum.

**Comment by Osman06TR** [comment]
Ok
Thanks
